### PR TITLE
Bugfix + trapezoidal absorber planes in LAr calo

### PR DESCRIPTION
**Pull request by gartrog**
@BrieucF

### CODE REVIEW

File: Detector/DetFCCeeECalInclined/compact/FCCee_ECalBarrel.xml [hunk 1]
```diff
@@ -19,6 +19,15 @@
...
   </info>
 
+  <materials>
+    <material name="LKr">
+      <!--<D value="1.396" unit="g/cm3" />-->
+      <D value="2.416" unit="g/cm3" />
+      <composite n="1" ref="Kr"/>
+    </material>
+  </materials>
+
```
Comment (BrieucF) [inline Detector/DetFCCeeECalInclined/compact/FCCee_ECalBarrel.xml:29]: Can't we put this in `DetCommon/compact/materials.xml`?

**Review by BrieucF — COMMENTED** [review]
Even though our workflow produces the calibration/dead material corrections on the flight, I would prefer that you also apply those changes to `Detector/DetFCCeeECalInclined/compact/FCCee_ECalBarrel_deadMaterial.xml` so that everything is consistent in the central repository. Otherwise it looks good to me. Thanks!

File: Detector/DetFCCeeECalInclined/compact/FCCee_ECalBarrel.xml (hunk 1 above)
Comment (gartrog) [reply Detector/DetFCCeeECalInclined/compact/FCCee_ECalBarrel.xml]: Yeah, done.

**Comment by gartrog** [comment]
> Even though our workflow produces the calibration/dead material corrections on the flight, I would prefer that you also apply those changes to `Detector/DetFCCeeECalInclined/compact/FCCee_ECalBarrel_deadMaterial.xml` so that everything is consistent in the central repository. Otherwise it looks good to me. Thanks!

Done.

**Comment by BrieucF** [comment]
Ha, this breaks the FCC-hh detectors [link](https://github.com/HEP-FCC/FCCDetectors/actions/runs/3154283150/jobs/5137637641#step:8:98)... Three solutions: 
- apply the change to the FCC-hh xml as well : https://github.com/HEP-FCC/FCCDetectors/tree/main/Detector/DetFCChhBaseline1/compact and https://github.com/HEP-FCC/FCCDetectors/tree/main/Detector/DetFCChhECalInclined/compact
- put a switch in the cpp geometry builder
- create a new geometry builder only for FCC-ee which will be completely factorized from FCC-hh

**Comment by gartrog** [comment]
At least for now I'd correct the FCChh xml files.

When we start to change more things, e.g more detailed cryostat, it would probably make sense to just create a FCCee specific geometry builder.

**Comment by BrieucF** [comment]
Agreed!

**Comment by gartrog** [comment]
Done

**Pull request merged**